packages/dx-react-grid-bootstrap3/src/templates/table-edit-cell.jsx

```diff
@@ -3,31 +3,42 @@ import * as PropTypes from 'prop-types';
 
 export const EditCell = ({
   column, value, onValueChange, style, children,
-  row, tableRow, tableColumn, editingEnabled, ...restProps
-}) => (
-  <td
-    style={{
-      verticalAlign: 'middle',
-      padding: '1px',
-      ...style,
-    }}
-    {...restProps}
-  >
-    {children || (
-      <input
-        type="text"
-        className="form-control"
-        value={value}
-        onChange={e => onValueChange(e.target.value)}
-        readOnly={!editingEnabled}
-        style={{
-          width: '100%',
-          textAlign: tableColumn && tableColumn.align,
-        }}
-      />
-    )}
-  </td>
-);
+  row, tableRow, tableColumn, editingEnabled,
+  autoFocus, onBlur, onFocus, ...restProps
```

Comment:
Why we don't get `onKeyDown` property from `props` here? I think it would be clear to understanding.